### PR TITLE
Display uploaded excel sheet

### LINKED ISSUE #10: xlsx to table

### PR DESCRIPTION
**Pull request by dittmanndennis**
closes #10 #20 #26

### CODE REVIEW

**Review by bcri25 — CHANGES_REQUESTED** [review]
hey

File: src/drag_drop/table/Table.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+      { field: 'roll' },
+      { field: 'pitch' },
+      { field: 'yaw' },
+      { field: 'robot', cellEditor: 'agSelectCellEditor', singleClickEdit: true, cellEditorParams: { values: robots } }
+  ];
+
+  const rowData = props.data;
+
+  function handleOnSubmit(){
+    console.log(gridRef.current.api.getDataAsCsv());  // replace with backend endpoint that provides error message for invalid input
```
Comment (dolevi-tub) [inline src/drag_drop/table/Table.js:32]: Even this waits for the backend integration:

See above: Don't use refs.
See below: Suggestion for getting this done using the library's own API without refs.

**Review by dolevi-tub — CHANGES_REQUESTED** [review]

File: src/drag_drop/table/Table.js (hunk 1 above)
Comment (dittmanndennis) [reply src/drag_drop/table/Table.js:32]: documentation suggests it with refs. https://ag-grid.com/react-data-grid/csv-export/

Comment (dolevi-tub) [reply src/drag_drop/table/Table.js]: I see less of a suggestion and more of an example on the `ag-grid-react` docs. And where examples go against `react` docs I'd prefer deviating from the examples

File: src/drag_drop/Drag.js [hunk 2]
```diff
@@ -1,51 +1,73 @@
...
+                    onDragEnter={() => updateBorder('over')}
+                    onDragLeave={() => updateBorder('leave')}
+          >
+            {({ getRootProps, getInputProps }) => (
+              <div {...getRootProps({ className: 'drop-zone' })} ref={dropRef}>
+                <input {...getInputProps()} />
+                <p>Drag and drop a file OR click here to select a file.</p>
+                {file && (
+                  <div>
+                    <strong>Selected file:</strong> {file.name}
```
[the remark this answers is not in the document]
Comment (dolevi-tub) [reply src/drag_drop/Drag.js]: Why is the explanatory string more prominent than the subject of the explanation?

**Pull request closed without merging**